pkg/controller/hostpathprovisioner/contoller_test.go

```diff
@@ -88,16 +88,20 @@ var _ = Describe("Controller reconcile loop", func() {
 				Namespace: "test-namespace",
 			},
 		}
+		dsNN := types.NamespacedName{
+			Name:      "hostpath-provisioner",
```

Comment:
Use the const you just create in common.go?

Reply:
Done